scripts/test.js

```diff
@@ -73,6 +49,7 @@ function test(rebuild, browsersArg) {
         console.log(data.toString());
     });
 
+    if (link == true) linkDependencies();

```

Comment:
Why you are comparing it with `true`? Isn't it supposed to be boolean?

Reply:
Because arguments[linkOptionIdx+1] returns string. But you are correct that the argument must be boolean - converted the command line argument to bool.